code/drasil-example/glassbr/lib/Drasil/GlassBR/Unitals.hs

```diff
@@ -102,8 +104,7 @@ standOffDist = uq (constrained' (uc sD (variable "SD") Real metre)
   [ gtZeroConstr,
     sfwrRange $ Bounded (Inc, sy sdMin) (Inc, sy sdMax)] (exactDbl 45)) defaultUncrt
 
-nomThick = cuc' "nomThick"
-  (nounPhraseSent $ S "nominal thickness t is in" +:+ eS (mkSet Rational (map dbl nominalThicknesses)))
+nomThick = cuc' "nomThick" (nounPhraseSP "nominal thickness")
```

Comment:
I'm fine with this change, but I wonder if you intended to commit it to this PR?